### PR TITLE
Hotjar - fix: add identify info

### PR DESCRIPTION
**Pull request by akashrpo**
## Description of the change

> updated the identify docs as per the js-sdk code

## Type of change
- [x] fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/analytics/hotjar.mdx [hunk 1]
```diff
@@ -39,15 +39,32 @@ Please follow our guide on <a href="https://rudderstack.com/docs/connections/add
...
-- Please provide the `Site ID`. You will get your `Site ID` under **Settings** - **Sites & Organizations** after logging into your Hotjar account.
-- You can enable or disable native SDK to send your events as per your preference. Finally, click on **Next** to complete the configuration.
+- Please provide the `Site ID`. You will get your `Site ID` under **Settings** - **Sites & Organizations** after logging into your Hotjar account. Finally, click on **Next** to complete the configuration.
 
 Hotjar will now be added and enabled as a destination in RudderStack.
 
-## Sending events to Hotjar
+## Identify
 
-There is **no need** to call any of `identify`, `page` or `track`methods explicitly, as Hotjar auto-tracks your user data.
```
Comment (saikumarrs) [inline destinations/analytics/hotjar.mdx:49]: I would assume Hotjar still auto-tracks data. So, let's keep this statement excluding `identify`?

File: destinations/analytics/hotjar.mdx [hunk 2]
```diff
@@ -39,15 +39,32 @@ Please follow our guide on <a href="https://rudderstack.com/docs/connections/add
...
+<div class="warningBlock">
+
+Before you can begin making calls to `identify`, User Attributes must be enabled in your Hotjar dashboard for each site you want to User Attributes within your account.
```
Comment (saikumarrs) [inline destinations/analytics/hotjar.mdx:50]: Can we add a link to their docs if available?

Comment (saikumarrs) [inline destinations/analytics/hotjar.mdx:50]: I've redacted the redundant parts. Feel free to rephrase to something better.
```suggestion
Before you can begin making calls to `identify`, `User Attributes` must be enabled in your Hotjar dashboard for each site.
```

File: destinations/analytics/hotjar.mdx [hunk 3]
```diff
@@ -39,15 +39,32 @@ Please follow our guide on <a href="https://rudderstack.com/docs/connections/add
 
 <img src="../../assets/image (42).png" /><span class="imageTitle">Hotjar Connection Settings in RudderStack</span>
```
Comment (saikumarrs) [inline destinations/analytics/hotjar.mdx:40]: The image seems to be obsolete as it's a device-mode only destination. Let's update it.

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: destinations/analytics/hotjar.mdx (hunk 3 above)
Comment (ameypv-rudder) [reply destinations/analytics/hotjar.mdx:40]: All the event stream destination images are being updated - ETA is next week.

File: destinations/analytics/hotjar.mdx (hunk 1 above)
Comment (akashrpo) [reply destinations/analytics/hotjar.mdx:49]: reverted

File: destinations/analytics/hotjar.mdx (hunk 2 above)
Comment (akashrpo) [reply destinations/analytics/hotjar.mdx]: added link to `User Attributes`

Comment (akashrpo) [reply destinations/analytics/hotjar.mdx]: seems good. rephrased it

File: destinations/analytics/hotjar.mdx (hunk 3 above)
Comment (akashrpo) [reply destinations/analytics/hotjar.mdx]: updated the connection settings image

File: destinations/analytics/hotjar.mdx (hunk 2 above)
Comment (saikumarrs) [reply destinations/analytics/hotjar.mdx]: Let's add this article reference that explains the process

Comment (akashrpo) [reply destinations/analytics/hotjar.mdx]: added the article reference for `User Attributes`

**Review by saikumarrs — APPROVED** [review]

**Review by utsabc — APPROVED** [review]
Looks good

**Review by ameypv-rudder — APPROVED** [review]
Standardized the content and made some formatting fixes.

**Pull request merged**